modules/backend/models/brandsetting/fields.yaml

```
@@ -106,4 +106,4 @@ tabs:
             type: codeeditor
             tab: backend::lang.branding.styles
             size: giant
-            language: css
+            language: scss
```

```suggestion
            language: less
```
Looking at the code in BrandSetting.php the language in use is actually LESS. Technically it could be made to support any form of CSS compilation supported by the Asset Compiler but we'll just leave it set to LESS for now.